Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -315,6 +319,26 @@ extension ActivityStreamPanel {
         let deleted = profile.prefs.arrayForKey(DefaultSuggestedSitesKey) as? [String] ?? []
         return suggested.filter({deleted.indexOf($0.url) == .None})
     }
+
+    func longPress(longPressGestureRecognizer: UILongPressGestureRecognizer) {
+        if longPressGestureRecognizer.state == UIGestureRecognizerState.Began {
+            let touchPoint = longPressGestureRecognizer.locationInView(self.view)
+            if let indexPath = tableView.indexPathForRowAtPoint(touchPoint) {
+                if indexPath.section == 1 {
```

Comment:
Ah very smart! I was worried that this would interfere with the TopSite long press. I'd use the Section enum here instead though instead of just "1"